src/wasm-interpreter.h

```diff
@@ -1376,6 +1378,7 @@ template<typename GlobalManager, typename SubType> class ModuleInstanceBase {
     : public ExpressionRunner<RuntimeExpressionRunner> {
     ModuleInstanceBase& instance;
     FunctionScope& scope;
+    std::vector<Literal> sideStack;
```

Comment:
how about `pushPopStack` to clarify it is just for them?

Reply:
Done.